caabernathy/YourWord
# New scriptures now shown on a weekly schedule

Resolves issue #9 so new memorization scriptures are shown weekly instead of whenever memorization is marked as done.



**Test plan (required)**
(Tested on Simulator where dates can be changed by changing the device date.)
1. Change date to previous week.
2. Build and run app.
3. Change date to current week
4. Run app
5. Ensure that a new scripture is shown and the previous week's scripture now shows up in the Arvhive tab.
